.circleci/config.yml

```diff
@@ -3,25 +3,37 @@ cleanAWS: &cleanAWS
     steps:
     - checkout
 
-    - attach_workspace:
-        at: .
-
-    - run: ./scripts/aws_credentials.sh ${TYPE}
-
-    - run: ./ci-cleaner aws --access-key-id ${AWS_ACCESS_KEY_ID} --secret-access-key ${AWS_SECRET_ACCESS_KEY} --region ${AWS_DEFAULT_REGION}
+    - run:
+        name: Set AWS credentials
+        command: ./scripts/aws_credentials.sh ${TYPE}
 
+    - run:
+        name: Execute ci-cleaner for AWS
+        command: |
+            docker run --rm -ti \
+                quay.io/giantswarm/ci-cleaner:latest \
+                aws \
+                --access-key-id ${AWS_ACCESS_KEY_ID} \
+                --secret-access-key ${AWS_SECRET_ACCESS_KEY} \
+                --region ${AWS_DEFAULT_REGION}
 
 
 cleanAzure: &cleanAzure
     machine: true
     steps:
     - checkout
 
-    - attach_workspace:
-        at: .
-
-    - run: ./ci-cleaner azure --client-id ${AZURE_CLIENT_ID} --client-secret ${AZURE_CLIENT_SECRET} --tenant-id ${AZURE_TENANT_ID} --subscription-id ${AZURE_SUBSCRIPTION_ID} --location ${AZURE_LOCATION}
-
+    - run:
```

Comment:
For my curiosity, why don't we need to setup AWS credentials?

Reply:
We do, but I reformatted the step to give it a proper name

Reply:
For that, look at step 2 of `cleanAWS`

Reply:
Sorry, I meant _azure_ credentials

Reply:
All credentials are available in Circle's environment variables. For Azure, we can just make use of them in flags. For AWS, we distinguish between control plane account and tenant cluster account. See the `TYPE` variable being set down below. That's what `aws_credentials.sh` takes care of.